client/src/lib/sessions/api/session.ts

```diff
@@ -219,3 +220,21 @@ export const acceptHostingInvite = async (
 
   return response.json();
 };
+
+export const getFeedbackCountByExercise = async (
+  exerciseId: string,
+  mode: SessionMode,
+): Promise<{positive: number; negative: number}> => {
+  const response = await apiClient(
+    `${SESSIONS_ENDPOINT}/exercise/${exerciseId}/${mode}/feedback/count`,
```

Comment:
I beleive this should be plural `exercises` as with `sessions`. May I suggest this instead:
`sessions/exercises/${exerciseId}/rating?mode=${mode}`?

The reason to put `mode`as a parameter is that it's param (like `limit`) rather than a resource and will probably be optional in the future.